### ISSUE TITLE
Update React to v0.14

### ISSUE THREAD

**Issue by kohei-takata** [workshopper/learnyoureact#55]

**Comment by shuhei** [comment]
The following made it work with React 0.13.3 for now. I had to install `react` by myself because npm 3 doesn't install peerDependencies.

``` sh
npm install express body-parser express-react-views@0.8.2 node-jsx react@0.13.3
```

**Comment by shuhei** [comment]
Ah, sorry. I didn't even need to use React 0.13.3. The following just worked.

``` sh
npm install express body-parser express-react-views node-jsx react react-dom
```

**Comment by kohei-takata** [comment]
:+1:

**Comment by kohei-takata** [comment]
fixed #57
